SECURITY.md

```diff
@@ -53,6 +53,86 @@ Here is the security disclosure policy for Node.js
   the release process above to ensure that the disclosure is handled in a
   consistent manner.
 
+## What does Node.js trust/not trust
+
+Being able to cause a negative outcome in a way that requires control of the
+elements that are trusted by Node.js is not considered a vulnerability in
+Node.js.
```

Comment:
This sentence and the similar sentences below are difficult to read/understand, at least for me. I think it might be the passive voice "considered". (Considered by whom? What's the difference between "is X" and "is considered X"? In general, there is/was a lot of "is considered deprecated" and "is considered X" and I try to get rid of it. "X is considered deprecated" is less clear than "X is deprecated".)

Reply:
Looking at this content, I do wonder if it's more appropriate in the HackerOne policy doc. Regardless, I wonder if it would be better formulated as something like this:

```md
Based on our threat model, Issues that we consider valid Node.js security concerns include:

* Lorem ipsum
* Lorem ipsum
* Lorem ipsum

Issues that we do not valid Node.js security concerns include:

* Lorem ipsum
* Lorem ipsum
* Lorem ipsum
```

Reply:
Honestly, I think the "Issues that are/aren't valid" is still confusing. @mhdawson any idea?

Reply:
The goal was to avoid having to write out a full least which I think is impractical.  We are trying to express that we trust certain things and if an attack involves something malicious/wrong in those things then its not valid.

Reply:
The purpose of this sentence is to identify what is out of scope for security researchers seeking a bug bounty. Is that basically correct?

Reply:
Basically, yes. Besides that, is for developers to understand when they are facing a bug or not.

Reply:
Non-blocking (because someone can always do this after the document lands), but in that case, might we want to replace this sentence with something like this?:

```suggestion
In the Node.js threat model, there are trusted elements such as the
underlying operating system. Vulnerabilities that require the compromise
of these trusted elements are outside the scope of the Node.js threat
model.

For a vulnerability to be eligible for a bug bounty, it must be a
vulnerability in the context of the Node.js threat model. In other
words, it cannot assume that a trusted element (such as the operating
system) has been compromised.
```

Reply:
Looks like a good change to me.